tests/python/tests.py

```diff
@@ -78,6 +79,36 @@ def test_admin_db_access():
     print(res)
     cleanup_conn(conn, cur)
 
+def test_idle_transaction_timeout():
+
+    # - - - - - - - - - - - - - - - - - -
+    # LESS THAN IDLE TRANSACTION TIMEOUT
+
+    conn, cur = connect_db(admin=False)
+    cur.execute("BEGIN;")
+    time.sleep(1)
+    cur.execute("SELECT 1;")
+    cur.execute("COMMIT;")
+    cleanup_conn(conn, cur)
+
+    # - - - - - - - - - - - - - - - - - -
+    # MORE THAN IDLE TRANSACTION TIMEOUT
+
+    conn, cur = connect_db(admin=False)
+    cur.execute("BEGIN;")
+    cur.execute("SELECT 1;")
+    time.sleep(IDLE_TRANSACTION_TIMEOUT + 1)
+
+    try:
+        cur.execute("SELECT 1;")
+    except psycopg2.OperationalError as e:
+        pass
```

Comment:
It would be good to check the error message and confirm you're getting the right one (`idle transaction timeout"`) and not something else.